core/internal/runhistory/runhistorysampler.go

```diff
@@ -0,0 +1,81 @@
+package runhistory
+
+import (
+	"encoding/json"
```

Comment:
should we use the simple encoding?

Reply:
alternatively we could use `"github.com/segmentio/encoding/json"`?

Reply:
in this case it doesn't matter, and in fact we can probably avoid using any JSON package at all. We're just parsing a string as a float.

Reply:
in case we decide to use something let's use github.com/segmentio/encoding/json
since it should be faster

Reply:
updated! i guess it's good for consistency as well

Reply:
yeah, probably should so search and replace. I think i saw some other place that was using the built-in encoding